### PR TITLE
IOSXE/show wireless mobility ap list

### PR DESCRIPTION
**Pull request by robschn**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    },
+    "b80-52-cap18": {
+        "controller_ip": "10.10.7.177",
+        "learnt_from": "Self",
+        "radio_mac": "58bf.ea37.1d60",
+    },
+    "b80-52-cap6": {
+        "controller_ip": "10.10.7.177",
+        "learnt_from": "Self",
+        "radio_mac": "58bf.ea13.75e0",
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py:15]: simply "ap_radio_mac" as same as output?

File: src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
+expected_output = {
+    "b80-22-cap10": {
```
Comment (tahigash) [inline src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py:2]: please have the key for values `b80-22-cap10` and others like:
```
    'ap_name': {
        "b80-22-cap10": {
            "controller_ip": "10.10.7.177",
            (snip)

**Review by tahigash — CHANGES_REQUESTED** [review]

File: src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py (hunk 1 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py]: updated!

File: src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py (hunk 2 above)
Comment (robschn) [reply src/genie/libs/parser/iosxe/tests/ShowWirelessMobilityApList/cli/equal/golden_output_expected.py]: updated!

**Comment by tahigash** [comment]
@badalsoyantar schema looks good now. Please review code itself.

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+        # b80-72-cap30                    58bf.eab3.1420    10.10.7.177      Self
+        ap_info_capture = re.compile(
+            r"^(?P<ap_name>\S+)\s+(?P<ap_radio_mac>\S{4}\.\S{4}\.\S{4})\s+(?P<controller_ip>\d+\.\d+\.\d+\.\d+)\s+(?P<learnt_from>\S+)\s+$"
+        )
+
+        ap_info_obj = {}
+
+        for line in output.splitlines():
+
```
[the remark this answers is not in the document]
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py:64]: Will add!

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+# ===================================
+# Schema for:
+#  * 'show wireless mobility ap-list'
+# ===================================
+class ShowWirelessMobilityApListSchema(MetaParser):
+    """Schema for show wireless mobility ap-list."""
+
+    schema = {Any(): {"controller_ip": str, "learnt_from": str, "radio_mac": str}}
```
[the remark this answers is not in the document]
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py]: I updated this in the newer commit

File: src/genie/libs/parser/iosxe/show_wireless.py [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+            r"^(?P<ap_name>\S+)\s+(?P<radio_mac>\S+)\s+(?P<controller_ip>\d+\.\d+\.\d+\.\d+)\s+(?P<learnt_from>\S+)"
+        )  # could also do (?P<learnt_from>(Self))
+
+        ap_info_obj = {}
+
+        for line in output.splitlines():
+
+            if ap_header_capture.match(line):
+                continue
+            elif delimiter_capture.match(line):
```
[the remark this answers is not in the document]
Comment (robschn) [reply src/genie/libs/parser/iosxe/show_wireless.py:67]: I'll take it off and see if it passes

**Review by badalsoyantar — APPROVED** [review]

**Review by choudnik — APPROVED** [review]

**Review by tahigash — APPROVED** [review]

**Comment by ThomasJRyan** [comment]
Please resolve merge Please resolve merge conflicts

**Comment by robschn** [comment]
> Please resolve merge Please resolve merge conflicts

Fixed!

**Pull request merged**